G/GraphicsMagick/build_tarballs.jl

```diff
@@ -70,8 +74,9 @@ dependencies = [
     # For OpenMP we use libomp from `LLVMOpenMP_jll` where we use LLVM
     # as compiler (BSD systems), and libgomp from
     # `CompilerSupportLibraries_jll` everywhere else.
-    Dependency("CompilerSupportLibraries_jll"; platforms=filter(!Sys.isbsd, platforms)),
-    Dependency("LLVMOpenMP_jll"; platforms=filter(Sys.isbsd, platforms)),
+    # Dependency("CompilerSupportLibraries_jll"; platforms=filter(!Sys.isbsd, platforms)),
+    # Dependency("LLVMOpenMP_jll"; platforms=filter(Sys.isbsd, platforms)),
+    Dependency("CompilerSupportLibraries_jll"),
```

Comment:
Is CSL needed at all if you disable openmp?

Reply:
I usually always add it as a safe thing! I can remove it.

Reply:
If the auditor doesn't say you should add then there's no need to add it.